Gems/Multiplayer/Code/Source/Components/NetworkHitVolumesComponent.cpp

```diff
@@ -222,4 +236,59 @@ namespace Multiplayer
     {
         m_actorComponent = nullptr;
     }
-}
+
+    void NetworkHitVolumesComponent::DrawDebugHitVolumes()
+    {
+        if (m_debugDisplay == nullptr)
```

Comment:
minor: I'd be nice to set `m_debugDisplay` and `m_physicsCharacter` to `nullptr` in `OnDeactivate`.

Reply:
Done